cmd/geth/config.go

```diff
@@ -24,6 +24,7 @@ import (
 	"reflect"
 	"unicode"
 
+	"github.com/ethereum/go-ethereum/rpc"
```

Comment:
Can you merge these two with other external imports in the same block?

Reply:
fix